quipucords/api/source/model.py

```diff
@@ -83,7 +83,7 @@ class Source(models.Model):
         SourceOptions, null=True, on_delete=models.CASCADE)
     credentials = models.ManyToManyField(Credential)
     hosts = models.TextField(unique=False, null=False)
-    exclude_hosts = models.TextField(unique=False, null=True)
+    exclude_hosts = models.TextField(unique=False, null=False)
```

Comment:
I'm pretty sure this should be set to `null=True`.

Reply:
If it is set to true the server returns 
`django.db.utils.IntegrityError: NOT NULL constraint failed: api_source.exclude_hosts`
after running a `qpc source add .... ` (This isn't covered in the code coverage tests)
From django model reference https://docs.djangoproject.com/en/2.0/ref/models/fields/:  
If null=True, Django will store empty values as NULL in the database. Default is False.